LLVMExports.cmake  contains absolute paths to sandbox

```
[cmake] CMake Error at /data/vchuravy/builds/julia/usr/lib/cmake/llvm/LLVMExports.cmake:793 (message):
[cmake]   The imported target "llvm-tblgen" references the file
[cmake] 
[cmake]      "/workspace/destdir/tools/llvm-tblgen"
[cmake] 
[cmake]   but this file does not exist.  Possible reasons include:
[cmake] 
[cmake]   * The file was deleted, renamed, or moved to another location.
[cmake] 
[cmake]   * An install or uninstall procedure did not complete successfully.
[cmake] 
[cmake]   * The installation package was faulty and contained
[cmake] 
[cmake]      "/data/vchuravy/builds/julia/usr/lib/cmake/llvm/LLVMExports.cmake"
```

--- a/L/LLVM/LLVM_full@9.0.1/build_tarballs.jl
+++ b/L/LLVM/LLVM_full@9.0.1/build_tarballs.jl
@@ -2,5 +2,6 @@ version = v"9.0.1"
 
 include("../common.jl")
 
+
 build_tarballs(ARGS, configure_build(ARGS, version)...;
                      preferred_gcc_version=v"7", preferred_llvm_version=v"8")

--- a/L/LLVM/common.jl
+++ b/L/LLVM/common.jl
@@ -139,10 +139,11 @@ CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING=True)
 
 # Julia expects the produced LLVM tools to be installed into tools and not bin
 # We can't simply move bin to tools since on MingW64 it will also contain the shlib.
-CMAKE_FLAGS+=(-DLLVM_TOOLS_INSTALL_DIR=${prefix}/tools)
+CMAKE_FLAGS+=(-DLLVM_TOOLS_BINARY_DIR="\${LLVM_INSTALL_PREFIX}/tools")
+CMAKE_FLAGS+=(-DLLVM_TOOLS_INSTALL_DIR="\${LLVM_INSTALL_PREFIX}/tools")
 
 # Also build and install utils, since we want FileCheck, and lit
-CMAKE_FLAGS+=(-DLLVM_UTILS_INSTALL_DIR=${prefix}/tools)
+CMAKE_FLAGS+=(-DLLVM_UTILS_INSTALL_DIR="\${LLVM_INSTALL_PREFIX}/tools")
 CMAKE_FLAGS+=(-DLLVM_INCLUDE_UTILS=True -DLLVM_INSTALL_UTILS=True)
 
 # Include perf/oprofile/vtune markers

